Makefile

```diff
@@ -4,6 +4,7 @@ golangci_lint := github.com/golangci/golangci-lint/cmd/golangci-lint@v1.49.0
 .PHONY: test
 test:
 	@go test -v ./...
+	@(cd handler/fasthttp; go test -v ./...)
```

Comment:
adrian acks that this is not the neatest way, but still in POC mode and confident we an make make pretty